helm-charts/common/llm-uservice/README.md

```diff
@@ -0,0 +1,28 @@
+# llm-uservice
+
+Helm chart for deploying llm microservice.
+
```

Comment:
better to use bigger case LLM instead, https://en.wikipedia.org/wiki/Large_language_model